sources/packages/api/src/route-controllers/institution-locations/institution-locations.controller.ts

```diff
@@ -27,4 +32,26 @@ export class InstitutionLocationsController extends BaseController {
       data: location.data,
     };
   }
+
+  @Post()
+  async create(
+    @Body() payload: InstitutionLocationType,
+    @UserToken() userToken: IUserToken,
+  ): Promise<GetInstitutionLocationDto> {
```

Comment:
I believe that this method should return a Promise<number> since it is returning only the id.